.storybook/theme.js

```diff
@@ -0,0 +1,6 @@
+import { create } from '@storybook/theming';
+
+export default create({
```

Comment:
I'm not a fan of default exports. Does this need to be one?

Reply:
Didn't even think of it as those configuration files usually rely on the default export. But this shouldn't be the case for this file. I change it to `module.exports = {...}`. Let's wait for the build if it still works. If yes, I will merge.

Reply:
It works, will merge it. Thanks.